File: android/app/src/main/java/com/svgrendernativeformat/SvgImagePackage.java
package com.svgrendernativeformat;

import androidx.annotation.NonNull;

import com.facebook.react.ReactPackage;
import com.facebook.react.bridge.NativeModule;
import com.facebook.react.bridge.ReactApplicationContext;
import com.facebook.react.uimanager.ViewManager;

import java.util.ArrayList;
import java.util.List;

public class SvgImagePackage implements ReactPackage {
    @NonNull
    @Override
    public List<NativeModule> createNativeModules(@NonNull ReactApplicationContext reactApplicationContext) {
        return new ArrayList<>();
    }

    @NonNull
    @Override
    public List<ViewManager> createViewManagers(@NonNull ReactApplicationContext reactApplicationContext) {
        ArrayList<ViewManager> managers = new ArrayList<>();
        managers.add(new SvgImageViewManager());
        return managers;
    }
}

File: android/app/src/main/java/com/svgrendernativeformat/SvgImagePackage.kt
package com.svgrendernativeformat

import com.facebook.react.ReactPackage
import com.facebook.react.bridge.NativeModule
import com.facebook.react.bridge.ReactApplicationContext
import com.facebook.react.uimanager.ViewManager
import com.svgrendernativeformat.svg.SvgRendererManager

class SvgImagePackage : ReactPackage {
    override fun createNativeModules(reactApplicationContext: ReactApplicationContext): List<NativeModule> {
        return emptyList()
    }

    override fun createViewManagers(reactApplicationContext: ReactApplicationContext): List<ViewManager<*, *>> {
        return listOf(SvgImageViewManager(reactApplicationContext), SvgRendererManager(reactApplicationContext))
    }
}